geode-memcached/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: "${project.projectDir}/../gradle/publish.gradle"
+
+dependencies {
+  compile(platform(project(':boms:geode-all-bom')))
+  compile(project(':geode-core'))
+  distributedTestCompile(project(':geode-dunit'))
+  testCompile('org.mockito:mockito-core')
+  testCompile(project(':geode-junit'));
+  integrationTestCompile('net.spy:spymemcached')
+  integrationTestCompile(project(':geode-junit'));
```

Comment:
Minor, but I'd prefer these generally ordered as:

```
dependencies {
  compile(platform(project(':boms:geode-all-bom')))
  compile(project(':geode-core'))

  testCompile(project(':geode-junit'))
  testCompile('org.mockito:mockito-core')

  integrationTestCompile(project(':geode-junit'))
  integrationTestCompile('net.spy:spymemcached')

  distributedTestCompile(project(':geode-dunit'))
}
```
to be (semi-)consistent with the other dependency blocks throughout..

Also, the semicolons aren't necessary.